examples/webgl_materials_cubemap_dynamic.html

```diff
@@ -28,7 +28,7 @@
 
 			textureLoader.load( 'textures/2294472375_24a3b8ef46_o.jpg', function ( texture ) {
 
-				texture.mapping = THREE.UVMapping;
+				texture.encoding = THREE.sRGBEncoding;
```

Comment:
Should `TextureLoader` set this when loading `jpg`s? [We are already setting the `RGBFormat`](https://github.com/mrdoob/three.js/blob/dev/src/loaders/TextureLoader.js#L35).

Reply:
AFAIK, jpgs are not always sRGB encoded. However, they do not support transparency.

Reply:
But I would say it's more likely a jpg will be sRGB encoded that not...?

Reply:
Definitely.

Reply:
>But I would say it's more likely a jpg will be sRGB encoded that not...?

I would not say that.

Normal map, specular map, roughness map, metalness map, alpha map, displacement map, light map, ao map, transmission map should not be sRGB encoded. Only diffuse, environment, and emissive.

Reply:
I think you have mixed texture types and image formats.

You normally want to use lossless image formats like PNG for textures that save non-color values like normal maps.

Reply:
>You normally want to use lossless image formats like PNG for textures that save non-color values like normal maps.

The normal map in your water example is JPG.

Likewise the decal example -- and other three.js examples. We do not have control over what users do.

Reply:
BTW, any idea what is blocking this PR?

Reply:
Yep, these textures should be changed.